ViewModel/Hydraulics/GriddedData/GriddedImporterVM.cs

```diff
@@ -71,49 +67,13 @@ public void AddRow( string name, string path, double probability, bool isEnabled
             ListOfRows.Add(newRow);
         }
 
-        #region copy files
-
-        private void Copy(string sourceDirectory, string targetDirectory)
-        {
-            DirectoryInfo diSource = new DirectoryInfo(sourceDirectory);
-            DirectoryInfo diTarget = new DirectoryInfo(targetDirectory);
-            CopyAll(diSource, diTarget);
-        }
-
-        private void CopyAll(DirectoryInfo source, DirectoryInfo target)
-        {
-            Directory.CreateDirectory(target.FullName);
-
-            // Copy each file into the new directory.
-            foreach (FileInfo fi in source.GetFiles())
-            {
-                string newPath = Path.Combine(target.FullName, fi.Name);
-                fi.CopyTo(newPath, true);
-            }
-
-            // Copy each subdirectory using recursion.
-            foreach (DirectoryInfo diSourceSubDir in source.GetDirectories())
-            {
-                DirectoryInfo nextTargetSubDir =
-                    target.CreateSubdirectory(diSourceSubDir.Name);
-                CopyAll(diSourceSubDir, nextTargetSubDir);
-            }
-        }
-
-        private void CopyWaterSurfaceFilesToStudyDirectory(string path, string nameWithExtension)
-        {
-            string destinationFilePath = Connection.Instance.HydraulicsDirectory + "\\"+ Name + "\\" + nameWithExtension;
-            Copy(path, destinationFilePath);
-        }
-        #endregion
-
         #region validation
         private FdaValidationResult ValidateImporter()
         {
             FdaValidationResult vr = new FdaValidationResult();
             if (ListOfRows.Count != 8)
             {
-                vr.AddErrorMessage("Eight hydraulic files are required to import.");
+                vr.AddErrorMessage("Eight hydraulic directories are required to import.");
```

Comment:
I am going to nit pick the language a little. Would you mind changing this to "Eight subdirectories with hydraulic modeling for 8 events, one event in each subdirectory, are required to import."